hudi-client/hudi-client-common/src/test/java/org/apache/hudi/testutils/MetadataMergeWriteStatus.java

```diff
@@ -60,12 +60,12 @@ private static void mergeMetadataMaps(Map<String, String> mergeFromMap, Map<Stri
       if (!mergeToMap.containsKey(key)) {
         mergeToMap.put(key, "0");
       }
-      mergeToMap.put(key, addStrsAsInt(entry.getValue(), mergeToMap.get(key)));
+      mergeToMap.put(key, addStrsAsLong(entry.getValue(), mergeToMap.get(key)));
     }
   }
 
-  private static String addStrsAsInt(String a, String b) {
-    return String.valueOf(Integer.parseInt(a) + Integer.parseInt(b));
+  private static String addStrsAsLong(String a, String b) {
+    return String.valueOf(Long.parseLong(a) + Long.parseLong(b));
```

Comment:
whats this fix is about?

Reply:
I triaged this. 
looks like the event time metadata tracking could have been broken only if not for this fix. I am not sure if someone was ever using this feature only.

Reply:
Right, this is a test class so it may cause some test failures